### ISSUE TITLE
uwsgi 2.0.23 builds failed

### ISSUE THREAD

**Issue by masterginger** [conda-forge/uwsgi-feedstock#75]
### Comment:

Hi all, we were trying to install uwsgi 2.0.23 from conda to make be compatible with Python 3.12 and realized the builds for https://github.com/conda-forge/uwsgi-feedstock/pull/73 (which was supposed to publish uwsgi 2.0.23 to conda) were failing.

I took a quick look at the build failure and saw `RuntimeError: Failed to download or patch source. Please see build log for info`, while it looked like the source download succeeded:
```
Source cache directory is: /home/conda/feedstock_root/build_artifacts/src_cache
Downloading source to cache: uwsgi-2.0.23_0cafda0c16.tar.gz
Downloading https://pypi.io/packages/source/u/uWSGI/uwsgi-2.0.23.tar.gz
Success
```

Could someone take a moment to check and see what's happening? I really appreciate the help!

**Comment by masterginger** [comment]
I did some investigation and realized the issue was not in "download" but in "patch". There's a [patch file](https://github.com/conda-forge/uwsgi-feedstock/blob/main/recipe/patches/0001-This-adds-support-for-Python-3.12.patch) in uwsgi-feedstock which tried to make uwsgi work with python 3.12. I looked into the uwsgi (not uwsgi-feedstock) source,  the content of the patch was already part of the uwsgi 2.0.23 source code, as a result, the patch failed.

I removed [this field](https://github.com/conda-forge/uwsgi-feedstock/blob/1c6fb2e1b114a1fc6ba731d983944d92e15f8d4c/recipe/meta.yaml#L11) and the issue was gone when building locally (I still got a build error but I believe that's specific to my local Xcode build chain since the linux aarch 64 build (docker based) succeeded.
